features/prompt_memes_a/challenge_promptmeme_setup.feature

```diff
@@ -26,21 +26,24 @@ Feature: Prompt Meme Challenge
   
   Given I have Battle 12 prompt meme fully set up
     And I am logged in as a random user
-  When I go to the collections page
+  When all indexing jobs have been run 
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.